### DISCUSSION TITLE
RedMatic Addon

### DISCUSSION THREAD

**Discussion by andesse** [OpenCCU/OpenCCU#1865] [🇩🇪 Generelle Diskussionen]
Hallo, das RedMatic Addon für Raspberrymatic hängt seit einer langen Zeit bei den Versionen hinterher, einige contribs laufen schon nicht mehr. 

Hat jemand mal etwas gehört wie es dort weiter geht? Kann das jemand zur Not forken und auf einen neuen Stand bringen?

Von Sebastian hört man selbst nichts mehr wenn man ein Issue/Discussion auf macht. :(

**Comment by jens-maus** [comment]
Nein, mir ist leider auch nicht bekannt warum/weshalb/wieso es bei RedMatic nicht mehr weitergeht und Sebastian (@hobbyquaker) nicht mehr erreichbar ist. Da es sich bei RedMatic jedoch um ein OpenSource Projekt handelt, sollte sich doch hoffentlich jemand finden lassen der das ganze einfach mal forkt und entsprechend weiterentwickelt.
